docs/pages/desktop-access/getting-started.mdx

```diff
@@ -51,14 +51,12 @@ to your Windows system, and prepare it for passwordless access through Teleport.
 
 ### Import the Teleport root certificate
 
-Use `tctl` to export the Teleport user certificate authority:
+Export the Teleport user certificate authority by running the following from your Windows system:
 
 ```code
-$ tctl auth export --type=windows > teleport.cer
+curl 'https://teleport-proxy.example.com:443/webapi/auth/export?type=windows' > teleport.cer
```

Comment:
I noticed single quotes here, but not in the previous doc, and a leading `$` in the previous doc, but not here. Can we be consistent?

Reply:
Oh sorry
Should be fixed now